Makefile

```diff
@@ -16,27 +16,16 @@
 ## Copyright (C) 2016  Dirk Eddelbuettel
 ## and released under GPL (>=2 )
 
-
-## check with https://github.com/adobe-fonts/source-code-pro/releases for version
-srcfile1=https://github.com/mozilla/Fira/archive/4.202.tar.gz
-outfile1=$(shell basename ${srcfile1})
-srcfile2=https://github.com/mozilla/Fira/archive/4.106.tar.gz
-outfile2=$(shell basename ${srcfile2})
-
+ARCHIVE		= master.zip
+ARCHIVE_LINK	= https://github.com/bBoxType/FiraSans/archive/$(ARCHIVE)
+UNPACKED_DIR	= FiraSans-master
 
 all:
-	test -f ${outfile1} || wget ${srcfile1} 
-	tar xfz ${outfile1}
+	wget ${ARCHIVE_LINK}
+	unzip ${ARCHIVE}
 	mkdir otf ttf
-	mv Fira-*/otf/* otf/
-	mv Fira-*/ttf/* ttf/
-	rm -rf Fira-* 
-	test -f ${outfile2} || wget ${srcfile2} #
-	tar xfz ${outfile2}
-	mv Fira-*/otf/*  otf/
-	mv Fira-*/ttf/*  ttf/
-	rm -rf Fira-* 
+	find ${UNPACKED_DIR} -name '*.otf' -exec cp {} otf/ \;
+	find ${UNPACKED_DIR} -name '*.ttf' -exec cp {} ttf/ \;
```

Comment:
Your changes are elegant and clean, yet maybe I would rather just stick with what obviously worked before ...